graders/r/plr/README.md

```diff
@@ -6,12 +6,11 @@ Helper functions for [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn
 [STAT430](https://stat430.com) (during 2018-2020) and now [STAT447](https://stat447.com) along
 with [STAT385](https://stat385.org).
 
-
 ### Example Workflow Within the PrairieLearn External Autograder
 
 Here we assume the instructor has created a file called `solution.R` in the `tests` directory
 containing the reference answer and that the student has submitted a file called `submission.R`.
-See [the PrairieLearn docs](https://prairielearn.readthedocs.io/en/latest/) for more details 
+See [the PrairieLearn docs](https://prairielearn.readthedocs.io/en/latest/) for more details
```

Comment:
This file didn't get checked in CI because it isn't on the `.dockerignore` allowlist, but it did get reformatted locally when I ran `make format-js`.